Tools/Frame_params/eLAB_LAB445_AC34.param

```diff
@@ -16,8 +16,8 @@ BATT_CAPACITY,3500
 BATT_MONITOR,3
 BATT_VOLT_MULT,10
 BRD_SAFETYENABLE,0
-CH7_OPT,3
```

Comment:
We shouldn't update these parameter files meant for older versions of the firmware.  they should be left as-is.

Reply:
We should remove these from "old configs" from our source tree IMO.... we should pull different versions of them from branches as needed...